src/main/kotlin/Main.kt

```diff
@@ -1,7 +1,30 @@
+import java.util.*
+
 fun main(args: Array<String>) {
-    println("Hello World!")
 
-    // Try adding program arguments via Run/Debug configuration.
-    // Learn more about running applications: https://www.jetbrains.com/help/idea/running-applications.html.
-    println("Program arguments: ${args.joinToString()}")
+    while (true) {
+        val n =  promtDouble("Введите десятичное число: ")
+        if (n == 0) {
+            break
+        }
+        displayln(binary(n))
+    }
+}
+
+fun display(text: String) {
+    print(text)
+}
+
+fun displayln(text: String) {
+    println(text)
+}
+
+fun promtDouble(text: String) : Int {
+    val reader = Scanner(System.`in`)
+    display(text)
+    return reader.nextInt()
+}
+
+fun binary (n: Int): String {
```

Comment:
После названия функции не ставят пробел, убери плиз
```suggestion
fun binary(n: Int): String {
```